packages/playground/vue/package.json

```diff
@@ -18,6 +18,7 @@
     "less": "^4.1.2",
     "pug": "^3.0.2",
     "sass": "^1.43.4",
-    "stylus": "^0.55.0"
+    "stylus": "^0.55.0",
+    "css-with-exports-field": "^1.0.0"
```

Comment:
I created a package in order to test this fix: https://unpkg.com/browse/css-with-exports-field@1.0.0/